.github/workflows/ci-dgraph-tests.yml

```diff
@@ -60,9 +60,15 @@ jobs:
           # build the test binary
           cd t; go build .
           # run the tests
-          ./t --coverage=true
+          ./t --coverage=true --skip=systest/acl/restore,systest/multi-tenancy,systest/backup/minio
           # clean up docker containers after test execution
           ./t -r
+          ./t --pkg=systest/acl/restore
+          ./t -r
+          ./t --pkg=systest/multi-tenancy
+          ./t -r
+          ./t --pkg=systest/backup/minio
+          ./t -r
```

Comment:
@matthewmcneely leaving notes for myself, this needs no action from your end.

cc @joshua-goldstein 

note: need to rebase

Reply:
rebasing is done